### ISSUE TITLE
"WARNING: unknown mimetype for [filename] ignoring" when running `make epub`

### ISSUE THREAD

**Issue by pdurbin** [sphinx-doc/sphinx#3214]
Is there any easy way to suppress "WARNING: unknown mimetype for [filename] ignoring" when running `make epub`? I'm using Sphinx 1.4.5.

I didn't see anything at http://sphinx.readthedocs.io/en/latest/faq.html#epub-info and from a quick look at https://github.com/sphinx-doc/sphinx/blob/1.5/sphinx/builders/epub.py#L587 I'm not sure what a fix would be.

Here's an example run of `make epub` from https://github.com/IQSS/dataverse/tree/adbe63d29dc3e8fd02593293d8f996659f36c7f6/doc/sphinx-guides

```
(dataverse) murphy:sphinx-guides pdurbin$ make clean
rm -rf build/*
(dataverse) murphy:sphinx-guides pdurbin$ make epub
sphinx-build -b epub -d build/doctrees   source build/epub
Running Sphinx v1.4.5
making output directory...
loading pickled environment... not yet created
loading intersphinx inventory from http://docs.python.org/objects.inv...
intersphinx inventory has moved: http://docs.python.org/ -> https://docs.python.org/2
building [mo]: targets for 0 po files that are out of date
building [epub]: targets for 55 source files that are out of date
updating environment: 55 added, 0 changed, 0 removed
reading sources... [100%] user/tabulardataingest/supportedformats                                                                                         
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] user/tabulardataingest/supportedformats                                                                                          
generating indices... genindex
writing additional pages... search
copying images... [100%] user/./img/Dataverse-Diagram.png                                                                                                 
copying static files... done
copying extra files... done
writing mimetype file...
writing META-INF/container.xml file...
writing content.opf file...
WARNING: unknown mimetype for _static/api/dataverse-complete.json, ignoring
WARNING: unknown mimetype for _static/api/dataverse-minimal.json, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/httpd/conf.d/dataverse.example.edu.conf, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/httpd/conf.d/ssl.conf, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/init.d/glassfish, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/init.d/solr, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/maintenance/maintenance.xhtml, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/selinux/targeted/src/policy/domains/misc/shibboleth.te, ignoring
WARNING: unknown mimetype for _static/installation/files/etc/shibboleth/shibGroupTestShib.json, ignoring
WARNING: unknown mimetype for _static/installation/files/home/rpmbuild/rpmbuild/RPMS/x86_64/rapache-1.2.7-rpm0.x86_64.rpm, ignoring
WARNING: unknown mimetype for _static/installation/files/issues/2180/grizzly-patch/glassfish-grizzly-extra-all.jar, ignoring
WARNING: unknown mimetype for _static/installation/files/issues/2180/grizzly-patch/readme.md, ignoring
writing toc.ncx file...
writing Dataverse.epub file...
build succeeded, 12 warnings.

Build finished. The epub file is in build/epub.
(dataverse) murphy:sphinx-guides pdurbin$ 
```

This was originally reported at https://github.com/IQSS/dataverse/issues/3391

**Comment by tk0miya** [comment]
Unfortunately there are no way to suppress them.
We should provide options to do that.

**Comment by tk0miya** [comment]
Fixed at 6d900c3.
Thank you for reporting!

**Comment by pdurbin** [comment]
@tk0miya wow! That was fast! Thanks for the bug fix! 😄
